docs/governance/justice/court-services-branch/governance.md

```diff
@@ -0,0 +1,201 @@
+
+**Specifications** are documents containing any combination of Machine-Testable Requirements and Human-Auditable Requirements needed to produce technical interoperability.
+
+## 1.3. Localization
+
+The standard language for this Governance Framework (GF) is English.
+
+## 1.4 Governing Authority
+
+The Chief Administrator of Court Services: ADM, Court Services Branch, Ministry of the Attorney General is the governing authority responsible for this Governance Framework (GF). 
+
+* 	**Name:** Jenny Manton
+* 	**Title:** Chief Administrator of Court Services
+* 	**Organization:** Court Services Branch, Ministry of the Attorney General
+* 	**Email:** Jenny.Manton@gov.bc.ca
+
+## 1.5. Administering Authority
+
+The Chief Administrator of Court Services: ADM, Court Services Branch, Ministry of the Attorney General is the administering authority responsible for this Governance Framework (GF). 
+
+* 	**Name:** Jenny Manton
+* 	**Title:** Chief Administrator of Court Services
+* 	**Organization:** Court Services Branch, Ministry of the Attorney General
+* 	**Email:** Jenny.Manton@gov.bc.ca
+
+## 1.6 Purpose 
+
+The purpose of this Governance Framework is to describe the rules/policies/procedures for verifiable credential exchanges involving B.C. Court Services Branch (CSB) with the open global community. This description enables participants to understand agreed-upon standards, terminology, and processes that allow the community to interact with the CSB in a trusted manner. 
+
+## 1.7 Scope
+
+The Court Services Branch (CSB) is a participant in an open ecosystem and the focus of this framework is to describe the processes CSB uses for digital credential exchanges.
+
+## 1.8 Objectives 
+
+1) Outline legislative and administrative processes relevant to digital credential exchanges.
+2) Support digital business interaction/automation within the B.C. justice ecosystem.
+
+## 1.9 Principles 
+
+[The BC Public Service](https://www2.gov.bc.ca/gov/content/careers-myhr/about-the-bc-public-service/ethics-standards-of-conduct/corporate-values) has one overarching corporate value, __Integrity__, and 6 core corporate values: Curiosity, Service, Passion, Teamwork, Accountability, and Courage. __Integrity__ is placed above all the other values as a quality that affirms the [Standards of Conduct for the B.C. Public Service](https://www2.gov.bc.ca/gov/content/careers-myhr/about-the-bc-public-service/ethics-standards-of-conduct/standards-of-conduct).
+
+## 1.10 General Requirements
+N/A
+
+## 1.11. Revisions
+
+Version 1.0. 
+
+## 1.12. Extensions
+
+There are no extensions to this GF.  
+
+## 1.13. Schedule of Controlled Documents
+
+N/A
+
+# 2. Controlled Documents
+
+## 2.1. Glossary
```

Comment:
Maybe make Glossary plural, since more than one are listed.

Reply:
This is to follow the ToIP spec.